epub2sphinx/convert.py

```diff
@@ -55,12 +55,19 @@ def generate_rst(book, source_directory):
             # Write ReST content
             ch_file.write(rst_content)
 
+def generate_index(book, dest_dir):
     # Generate index.rst
-    generate_index(file_names)
-
-def generate_index(input_epub):
-    # Generate index.rst
-    pass
+    with open(f"{dest_dir}/source/index.rst", 'w') as f:
+        f.write(f"{book.title}\n")
+        f.write("==============================\n\n")
+        f.write(f"Author: {book.get_metadata('DC', 'creator')[0][0]}\n\n")
```

Comment:
I am not sure if Author is a mandatory attribute for an EPub file, and it will throw an error if the `creator` is empty.
Should I remove it?